### ISSUE TITLE
A problem occurred configuring root project 'audioplayers'. > SDK location not found. Define location with sdk.dir in the local.properties file or with an ANDROID_HOME environment variable

### ISSUE THREAD

**Issue by singlakaran** [luanpotter/audioplayers#763]
i got this error

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'audioplayers'.
> SDK location not found. Define location with sdk.dir in the local.properties file or with an ANDROID_HOME environment variable.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 811ms
```
Flutter Doctor:-
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.22.6, on macOS 11.2.1 20D74 darwin-x64, locale en-IN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 12.4)
[✓] Android Studio (version 4.0)
[✓] Connected device (1 available)
```

**Comment by jinfagang** [comment]
@singlakaran  Same issue here after upgrade to flutter:

```
Flutter 1.27.0-2.0.pre.98 • channel master • https://github.com/flutter/flutter.git
Framework • revision 696e4f70d1 (13 hours ago) • 2021-02-13 18:31:02 -0500
Engine • revision 1d537824d6
Tools • Dart 2.13.0 (build 2.13.0-30.0.dev)

Running flutter doctor...
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, 1.27.0-2.0.pre.98, on Microsoft Windows [Version 10.0.19042.746], locale zh-CN)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1.0)
[✓] VS Code (version 1.53.2)
[✓] Connected device (3 available)
```


Any ideas?

**Comment by shahparan71** [comment]
Same error. Any idea?

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'audio_recorder'.
> SDK location not found. Define location with sdk.dir in the local.properties file or with an ANDROID_HOME environment variable.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 6m 18s


Exception: The plugin audio_recorder could not be built due to the issue above.

**Comment by najib-mrh** [comment]
Same issue !

**Comment by bahcem** [comment]
I get the same error when add firebase_admob package. What is the solution for

**Comment by najib-mrh** [comment]
> I get the same error when add firebase_admob package. What is the solution for

in my case, i solved the problem by updating the gradle to 4.0.1 
https://android-developers.googleblog.com/2020/07/preparing-your-build-for-package-visibility-in-android-11.html

**Comment by erickzanardo** [comment]
This is usually a error regarding your Android setup, the log tells you that it can't find your SDK location:

```
> SDK location not found. Define location with sdk.dir in the local.properties file or with an ANDROID_HOME environment variable.
```
